documentation_quality_analysis/analyze_library/prioritize_api/llm/detect_api_discussion.py

```diff
@@ -12,25 +11,19 @@
 
 logging.basicConfig(level=logging.INFO)
 
-# load_dotenv()
-
 client = OpenAI(api_key=os.getenv("OPENAI_API_KEY"))
 
+
 @retry(wait=wait_random_exponential(min=1, max=60), stop=stop_after_attempt(2))
```

Comment:
so right now, you are trying twice if the api failed to provide a response?

Reply:
yes. I just gave 2 attempts now.